chat_shell/chat_shell/tools/builtin/silent_exit.py

```
@@ -0,0 +1,97 @@
+        "Call this tool when the execution result does not require user attention. "
+        "For example: regular status checks with no anomalies, routine data collection "
+        "with expected results, or monitoring tasks where everything is normal. "
+        "This will end the conversation immediately without notifying the user, "
+        "and the execution will be hidden from the timeline by default."
+    )
+    args_schema: type[BaseModel] = SilentExitInput
+
+    def _run(
+        self,
+        reason: str = "",
+        run_manager: CallbackManagerForToolRun | None = None,
```

Ruff (0.14.13)

Unused method argument: `run_manager`

(ARG002)